inventory/hosts

```diff
@@ -243,3 +244,22 @@ br
 co
 cr
 hu
+ch
+
+#------------------------------------------------------------------------------
+# Costa Rica
+
+[cr-prod]
+laferia.cr ansible_host=95.217.177.232
+
+[cr:children]
+cr-prod
+
```

Comment:
Why are you adding Costa Rica here, too? We have that already:
https://github.com/openfoodfoundation/ofn-install/blob/92258df9e070c695836a2ca49f721652218bffae/inventory/hosts#L66-L74